File: 백준/Bronze/11319. Count Me In/Count Me In.py
t = int(input())
vowels_list = ["A", "E", "I", "O", "U", "a", "e", "i", "o", "u"]
for _ in range(t):
    s = input()
    v_cnt = 0
    c_cnt = 0
    for i in range(len(s)):
        if s[i] in vowels_list:
            v_cnt += 1
        elif s[i] != " ":
            c_cnt += 1
    print(c_cnt, v_cnt)

File: 백준/Bronze/11319. Count Me In/Count Me In.java

import java.util.*;
import java.math.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt(); // nextInt() 마지막 개행문자 제거안해줌
        sc.nextLine();
        for (int i=0;i<t;i++) {
            String s = sc.nextLine();
            s = s.toLowerCase();
            int v_cnt = 0;
            int c_cnt = 0;
            
            for (int j=0;j<s.length();j++) {
                if (s.charAt(j)=='a' || s.charAt(j)=='e'||s.charAt(j)=='i'||s.charAt(j)=='o'||s.charAt(j)=='u'){
                    v_cnt += 1;
                }
                else if (s.charAt(j)!=' ') {
                    c_cnt += 1;
                }
            }
            System.out.println(c_cnt+" "+v_cnt);
        } //for testcase

    }
}

